docs/writing-controllers.md

```diff
@@ -1400,7 +1394,7 @@ import { accountsControllerSelectors } from '@metamask/accounts-controller';
 
 type AllowedActions = AccountsControllerGetStateAction;
 
-export type TokensControllerMessenger = RestrictedControllerMessenger<
```

Comment:
One more case where we might want to stick with TokensControllerMessenger. I think this is the only one left 😅

Reply:
Good catch, again! 🤦 

Done in 5dddb45fc